plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -204,25 +203,23 @@ fn init(cfg_dir: PathBuf) -> Result<(), anyhow::Error> {
 
 fn create_operation_files(config_dir: &str) -> Result<(), anyhow::Error> {
     create_directory_with_user_group(&format!("{config_dir}/c8y"), "root", "root", 0o775)?;
```

Comment:
According to the Didier's comment https://github.com/thin-edge/thin-edge.io/pull/1217#issuecomment-1165430950, the `/etc/tedge/c8y` directory permission also should be changed.

```suggestion
    create_directory_with_user_group(&format!("{config_dir}/c8y"), "root", "root", 0o1777)?;
```

Reply:
done